packages/api/src/server/analytics/service/analytics.service.ts

```diff
@@ -204,11 +212,16 @@ export class AnalyticsService {
   }
 
   async getDeploymentTimeSaved(): Promise<Object> {
-    const response = await this.getAverageDeploymentTime('', 'NA', AnalyticsService.defaultDays * AnalyticsService.days);
+    let response;
+    try {
+      response = await this.getAverageDeploymentTime('', 'NA', AnalyticsService.defaultDays * AnalyticsService.days);
```

Comment:
Hi @arkaprovob ,`getAverageDeploymentTime()` method is already throwing the Error if no data present. So If no data is coming the it will always go to catch, Because it's handled in the `getAverageDeploymentTime()` Function.